find mispriming regions

kriemo opened the pull request:
Added a utility to identify regions enriched for mispriming events. This needs some exploratory analysis to determine appropriate amount of flanking region to include in reported intervals.

kriemo committed:
add n_iter

kriemo committed:
add fxn to find mispriming regions

kriemo committed:
fix some compiler warnings

kriemo committed:
add param to doc

kriemo committed:
Merge branch 'main' into filter-mispriming

kriemo committed:
add news item

jayhesselberth commented:
It might be nice if this would accept multiple BAM files, to be able to generate one merged GR that can be cross-referenced with a final RSE.

kriemo committed:
sync w/ main

Merge branch 'main' into filter-mispriming

# Conflicts:
#	NAMESPACE
#	NEWS.md
#	R/filter_se.R
#	src/plp.c
#	src/plp_utils.c
#	src/sc-plp.c

kriemo committed:
add test

kriemo committed:
add returns to documentation

kriemo commented:
I'm going to merge in the current single bam approach for now. Adding support (or guidance) for how best to merge regions from multiple bams can be added later.
